### DISCUSSION TITLE
Choice to Toggle Visibility of Specific Guest Thoughts

### DISCUSSION THREAD

**Discussion by Derpercoaster** [OpenRCT2/OpenRCT2#17305] [Feature requests]
Sometimes when I'm looking through the list of guest thoughts (specifically in the "summarize" tab), I want to only see specific thoughts while ignoring others. Could it be made so thoughts you don't want to see can be toggled off?

**Comment by J-PIE-314** [comment]
This sounds like a good feature for a plugin.

**Reply by ZehMatt** [reply]
Plugins can not alter built-in Ui, do you perhaps mean that a plugin could provide a more in-depth guest list?

**Reply by J-PIE-314** [reply]
Yes, although I believe the OP is suggesting here that someone implement something similar to message filtering (as seen below).

![messages filters](https://github.com/user-attachments/assets/59f9ea3c-3ae9-4121-8db4-7a75122c96f9)
